### PR TITLE
Markup auth token secret

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: meta/255bae6f.csv [hunk 1]
```diff
@@ -337,3 +337,5 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 133085,f2450a1b,GitHub,255bae6f,data/255bae6f/src/f2450a1b.rb,115:115,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 133086,fb303b7e,GitHub,255bae6f,data/255bae6f/src/fb303b7e.rb,131:131,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 133087,fb303b7e,GitHub,255bae6f,data/255bae6f/src/fb303b7e.rb,137:137,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
+135000,873d5f9d,GitHub,255bae6f,data/255bae6f/src/873d5f9d.rb,16:16,T,F,11,19,F,F,Any,,,Secret,2.50,8,F,F,F,Password
+135001,873d5f9d,GitHub,255bae6f,data/255bae6f/src/873d5f9d.rb,10:10,T,F,11,19,F,F,Any,,,Secret,2.50,8,F,F,F,Password
```
Comment (babenek) [inline meta/255bae6f.csv:341]:
```suggestion

```

File: meta/57c424f8.csv [hunk 2]
```diff
@@ -51,3 +51,4 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 114560,b17149db,GitHub,57c424f8,data/57c424f8/src/b17149db.sql,479:479,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
 114561,b17149db,GitHub,57c424f8,data/57c424f8/src/b17149db.sql,483:483,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
 114562,b17149db,GitHub,57c424f8,data/57c424f8/src/b17149db.sql,484:484,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
+100007,f23d7ddd,GitHub,57c424f8,data/57c424f8/test/f23d7ddd.java,42:42,F,F,13,18,F,F,CharOnly,,,Secret,2.32,5,F,F,F,Password
```
Comment (babenek) [inline meta/57c424f8.csv:54]:
```suggestion

```

File: meta/6c73b80a.csv [hunk 3]
```diff
@@ -828,3 +828,5 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 134815,8e39a8bd,GitHub,6c73b80a,data/6c73b80a/test/8e39a8bd.go,3916:3916,T,F,40,47,F,F,,,Info,,0,0,F,F,F,Other
 134816,8e39a8bd,GitHub,6c73b80a,data/6c73b80a/test/8e39a8bd.go,3918:3918,T,F,40,47,F,F,,,Info,,0,0,F,F,F,Other
 134817,8e39a8bd,GitHub,6c73b80a,data/6c73b80a/test/8e39a8bd.go,3919:3919,T,F,41,48,F,F,,,Info,,0,0,F,F,F,Other
+134818,88763956,GitHub,6c73b80a,data/6c73b80a/test/88763956.go,737:737,F,T,26,31,F,F,Any,,,Secret,2.32,5,F,F,F,Password
+135002,3f79ca86,GitHub,6c73b80a,data/6c73b80a/test/3f79ca86.go,325:325,T,F,41,48,F,F,Any,,,Secret,2.52,7,F,F,F,Password
```
Comment (babenek) [inline meta/6c73b80a.csv:832]:
```suggestion
134818,3f79ca86,GitHub,6c73b80a,data/6c73b80a/test/3f79ca86.go,325:325,T,F,41,48,F,F,Any,,,Secret,2.52,7,F,F,F,Password

```

File: meta/7f0408de.csv [hunk 4]
```diff
@@ -235,3 +235,4 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 132071,5f0d7466,GitHub,7f0408de,data/7f0408de/other/5f0d7466.md,334:334,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 132072,ca04bea7,GitHub,7f0408de,data/7f0408de/test/ca04bea7.md,228:228,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 133489,abfe0f70,GitHub,7f0408de,data/7f0408de/other/abfe0f70.md,695:695,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
+133490,372e9b20,GitHub,7f0408de,data/7f0408de/other/372e9b20.md,201:201,F,T,12,17,F,F,CharOnly,,,Secret,2.32,5,F,F,F,Password
```
Comment (babenek) [inline meta/7f0408de.csv:238]:
```suggestion

```

File: meta/a09d9e50.csv [hunk 5]
```diff
@@ -99,7 +99,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 6980,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,2887:2887,F,F,,,F,F,,,,,0,0,F,F,F,Predefined Pattern
 6981,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,3036:3036,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 6982,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,3416:3416,F,F,,,F,F,,,,,0,0,F,F,F,Other
-6983,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,819:819,T,F,,,F,F,,,,,0,0,F,F,F,Authentication Credentials
+6983,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,819:819,T,F,538,669,F,F,,,,,0,0,F,F,F,Authentication Credentials
```
Comment (babenek) [inline meta/a09d9e50.csv:102]:
```suggestion
6983,054074fe,GitHub,a09d9e50,data/a09d9e50/test/054074fe.ndjson,819:819,T,F,538,669,F,F,,,Google OAuth Access Token,Auth,0,0,F,F,F,Authentication Credentials
```

**Review by babenek — COMMENTED** [review]
corrections

File: meta/c9b945fa.csv [hunk 6]
```diff
@@ -192,13 +192,13 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 114332,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,286:286,Template,F,56,59,F,F,CharOnly,,,Auth,0.92,3,F,F,F,Authentication Credentials
 114333,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,314:314,Template,F,56,59,F,F,CharOnly,,,Auth,0.92,3,F,F,F,Authentication Credentials
 131401,a1cb7340,GitHub,c9b945fa,data/c9b945fa/src/a1cb7340.py,212:212,F,F,,,F,F,,,,,0.00,,F,F,F,Password
-131402,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,149:149,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
+131402,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,149:149,F,F,161,191,F,F,,,,,0.00,,F,F,F,Authentication Credentials
 131403,165bee60,GitHub,c9b945fa,data/c9b945fa/src/165bee60.rst,155:155,F,F,,,F,F,,,,,0.00,,F,F,F,Password
-131404,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,110:110,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
+131404,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,110:110,F,F,161,191,F,F,,,,,0.00,,F,F,F,Authentication Credentials
```
Comment (babenek) [inline meta/c9b945fa.csv:197]:
```suggestion
131404,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,110:110,T,F,161,191,F,F,,,,,0.00,,F,F,F,Authentication Credentials
```

File: meta/c9b945fa.csv [hunk 7]
```diff
@@ -192,13 +192,13 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 114332,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,286:286,Template,F,56,59,F,F,CharOnly,,,Auth,0.92,3,F,F,F,Authentication Credentials
 114333,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,314:314,Template,F,56,59,F,F,CharOnly,,,Auth,0.92,3,F,F,F,Authentication Credentials
 131401,a1cb7340,GitHub,c9b945fa,data/c9b945fa/src/a1cb7340.py,212:212,F,F,,,F,F,,,,,0.00,,F,F,F,Password
-131402,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,149:149,F,F,,,F,F,,,,,0.00,,F,F,F,Authentication Credentials
+131402,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,149:149,F,F,161,191,F,F,,,,,0.00,,F,F,F,Authentication Credentials
```
Comment (babenek) [inline meta/c9b945fa.csv:195]:
```suggestion
131402,0cb23f01,GitHub,c9b945fa,data/c9b945fa/test/0cb23f01.py,149:149,T,F,161,191,F,F,,,,,0.00,,F,F,F,Authentication Credentials
```

File: meta/81cd05d0.csv [hunk 8]
```diff
@@ -5295,7 +5295,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 133795,c63bf4a5,GitHub,81cd05d0,data/81cd05d0/src/c63bf4a5.json,211:211,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 133796,c63bf4a5,GitHub,81cd05d0,data/81cd05d0/src/c63bf4a5.json,342:342,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 133797,f759ddf8,GitHub,81cd05d0,data/81cd05d0/src/f759ddf8.json,6862:6862,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
-133798,58013633,GitHub,81cd05d0,data/81cd05d0/test/58013633.java,122:122,F,F,,,F,F,,,,,0,0,F,F,F,Generic Token
+133798,58013633,GitHub,81cd05d0,data/81cd05d0/test/58013633.java,122:122,F,F,29,84,F,F,,,,,0,0,F,F,F,Generic Token
```
Comment (babenek) [inline meta/81cd05d0.csv:5298]:
```suggestion
133798,58013633,GitHub,81cd05d0,data/81cd05d0/test/58013633.java,122:122,T,F,29,84,F,F,,,,,0,0,F,F,F,Generic Token
```

**Review by babenek — COMMENTED** [review]
suggested changes

**Review by babenek — APPROVED** [review]
LGTM.

**Review by xDizzix — APPROVED** [review]

**Pull request merged**